packages/server/src/centrifuge-client/centrifuge.service.ts

```diff
@@ -36,8 +36,8 @@ export class CentrifugeService {
       if (result.status === 'pending') {
         return delay(250).then(() => this.pullForJobComplete(jobId, authorization));
       } else if (result.status === 'failed') {
-        console.log('Job Failed', result)
-        throw new BadRequestException(result.message);
+        console.log('Job Failed', result.message)
+        return result;
```

Comment:
why do you want to return a 200 when the result fails?

Reply:
because the document / mint status expects a result, which triggers a status update, it's not a 200 but a JobsStatusResponse containing the status from the node that i want to pass along